pkg/volume/plugins.go

```diff
@@ -71,8 +71,10 @@ const (
 
 var (
 	deprecatedVolumeProviders = map[string]string{
-		"kubernetes.io/cinder":  "The Cinder volume provider is deprecated and will be removed in a future release",
-		"kubernetes.io/scaleio": "The ScaleIO volume provider is deprecated and will be removed in a future release",
+		"kubernetes.io/cinder":    "The Cinder volume provider is deprecated and will be removed in a future release",
+		"kubernetes.io/storageos": "The StorageOS volume provider is deprecated and will be removed in a future release",
+		"kubernetes.io/quobyte":   "The Quobyte volume provider is deprecated and will be removed in a future release",
```

Comment:
Have we reached out to quobyte owners?

Reply:
Yep, this is their website doc PR: https://github.com/kubernetes/website/pull/27644/files
This is their response on the quobyte deprecation: https://github.com/kubernetes/kubernetes/issues/100900#issuecomment-823279294